pkgs/os-specific/linux/purism/librem-ec-acpi/default.nix

```diff
@@ -0,0 +1,41 @@
+{
+  lib,
+  stdenv,
+  fetchFromGitLab,
+  kernel,
+}:
+
+stdenv.mkDerivation rec {
```

Comment:
Nit: I'd drop the `rec` here, as I don't think there's currently anything recursive about your definition.